topology/probes/netlink.go

```diff
@@ -492,6 +540,23 @@ func getFamilyKey(family int) string {
 	return ""
 }
 
+func (u *NetNsNetLinkProbe) onRouteChanged(index int, route *RouteTable) {
+	u.Graph.Lock()
+	defer u.Graph.Unlock()
+
+	intf := u.Graph.LookupFirstChild(u.Root, graph.Metadata{"IfIndex": index})
```

Comment:
here I am getting nil. I have simple.sh script's topology. used this cmd to add a default route "sudo ip netns exec vm2 route add default gw 192.168.10.18 eth0"